mmv1/products/cgc/terraform.yaml

```diff
@@ -51,6 +51,13 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           deletion_protection: "false"
         ignore_read_extra:
           - "deletion_protection"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "sql_mysql_instance_authorized_network"
+        primary_resource_id: "instance"
+        vars:
+          mysql_instance_with_authorized_network: "mysql-instance-with-authoriz$
```

Comment:
Looks like there's a syntax error on this line!

Reply:
fixed